README.md

```diff
@@ -144,7 +144,7 @@ See [Speaker group management](#speaker-group-management) for example usage.
 | show_group_count | boolean | true | Have the number of grouped speakers displayed (if any) in the card name.
 | icon | string | optional | Override default group button icon *(any mdi icon)*.
 
-<a name="speaker_foot1"><sup>1</sup></a> Currently not yet supported in Home Assistant, *soon™*
+<a name="speaker_foot1"><sup>1</sup></a> Currently supported by a [custom component](https://github.com/ppanagiotis/pymusiccast) available in HACS
```

Comment:
Thanks, great work, I would prefer if it said something like "Requires custom component ..." instead.

Reply:
Done ;-)